plugins/techdocs/src/reader/transformers/sanitizeDOM.ts

```diff
@@ -34,14 +34,51 @@ export const safeLinksHook = (node: Element) => {
   return node;
 };
 
+const filterIframeHook = (allowedIframeHosts: string[]) => (node: Element) => {
+  if (node.nodeName === 'IFRAME') {
+    const src = node.getAttribute('src');
+    if (!src) {
+      node.remove();
+      return node;
+    }
+
+    try {
+      const srcUrl = new URL(src);
+      const isMatch = allowedIframeHosts.some(host => srcUrl.host === host);
```

Comment:
one thing that just came to mind, I wonder if we should allow hosts specified with wildcards in the domain. For example something like `.*.google.com` and then we can do something like 

```ts
allowedIframeHosts.some(host => srcUrl.host.match(host));
```

What do you think @Rugvip @dnsaoki2 ?

Reply:
@emmaindal I don't know if there would be any security issues, a point of attention would be to pass for example ".*" as allowed host, this will basically allow any iframe

Reply:
that is a very good point, thanks for brining that up! Let's not support that for now and keep it to the way it is!

Reply:
I'd say let's not add it unless it turns out we really need it tbh, bit too sensitive area. I'd also steer clear of regex for this kind of matching too, as for example `xyzabcgoogle.com` matches the example pattern